specs/phase0/beacon-chain.md

```diff
@@ -984,7 +987,7 @@ def get_beacon_proposer_index(state: BeaconState) -> ValidatorIndex:
     Return the beacon proposer index at the current slot.
     """
     epoch = get_current_epoch(state)
-    seed = hash(get_seed(state, epoch, DOMAIN_BEACON_PROPOSER) + int_to_bytes(state.slot, length=8))
+    seed = hash(get_seed(state, epoch, DOMAIN_BEACON_PROPOSER) + int_to_bytes(state.slot, length=uint64(8)))
```

Comment:
IMHO this really shouldn't be a uint64. An int can't be bigger than 32 bytes here, a uint64 for the length is weird. Which fits in every integer type. Can we just define it as `int` in the util method, since it doesn't have to be SSZ anyway?

Reply:
I think we can!

The `uint64` for `int_to_bytes` param is mainly for ensuring the input is non-negative, and we don't have non-negative number type in native Python. I'm okay to change it back to `int` since `uint64` might be "too much" here.